packages/at_client/lib/src/service/encryption_service.dart

```diff
@@ -316,6 +318,7 @@ class EncryptionService {
       throw KeyNotFoundException('private encryption key not found');
     }
     var sharedKey =
+        // ignore: deprecated_member_use_from_same_package
```

Comment:
instead of annotating in each place can you check whether there is an option for add lint rules in analysis_options.yaml

Reply:
verified with  deprecated_member_use_from_same_package: ignore
It still throws issues.